data/gegno/gegno outfits.txt

```diff
@@ -811,6 +811,49 @@ effect "bludgeon impact"
 	sound "bludgeon impact"
 
 
+
+# Hand to Hand Weapons
+
+outfit "Savagery Pike"
+	category "Hand to Hand"
+	cost 21000
```

Comment:
Fragmentation Grenades at 17k and do more attack (1.3 vs 0.9) and have less defense (0.3 vs 0.9), making them much better than this.
```suggestion
	cost 13000
```